cryptostock/account/tests/test_get_list_salesdashboard.py

```diff
@@ -59,15 +60,48 @@ def test_get_list_sales_dashboard_db_calls(auth_user, broker_account):
         response = auth_user.get("/api/salesdashboard/")
 
     assert response.status_code == 200
-    assert len(query_context) == 2
+    assert len(query_context) == 3
 
     SalesDashboardFactory.create_batch(1000, broker=broker_account)
 
     with CaptureQueriesContext(connection) as query_context:
         response = auth_user.get("/api/salesdashboard/")
 
     assert response.status_code == 200
-    assert len(query_context) == 2
+    assert len(query_context) == 3
+
+
+@pytest.mark.parametrize(
+    "count,limit,offset", [(50, 1, 0), (100, 10, 0), (1000, 10, 2)]
+)
+def test_get_list_sales_dashboard_pagination_check_limit(
+    auth_user, count, limit, offset
+):
+    SalesDashboardFactory.create_batch(count)
+
+    response = auth_user.get(f"/api/salesdashboard/?limit={limit}&offset={offset}")
+
+    assert response.status_code == 200
+    assert tuple(response.json().keys()) == ("count", "next", "previous", "results")
```

Comment:
Keys can have different order between tests. Use set:
```suggestion
    assert response.json().keys() == {"count", "next", "previous", "results"}
```

Reply:
got it